scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -58,12 +56,18 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
     toImport.asPatch + predefUnderscore
   }
 
-  val rangeImport = Symbol("_root_.strawman.collection.immutable.Range.")
-  val inclusiveRange = Symbol(
-    "_root_.scala.runtime.RichInt#to(I)Lscala/collection/immutable/Range/Inclusive;.")
-  val rangeSymbol = Symbol(
-    "_root_.scala.runtime.RichInt#until(I)Lscala/collection/immutable/Range;.")
+  def removeGlobalImports(ctx: RuleCtx): Patch =
+    ctx.removeGlobalImport(Symbol("_root_.scala.collection.")) +
+      ctx.removeGlobalImport(Symbol("_root_.scala.collection.concurrent.")) +
+      ctx.removeGlobalImport(Symbol("_root_.scala.collection.immutable.")) +
+      ctx.removeGlobalImport(Symbol("_root_.scala.collection.mutable."))
```

Comment:
even though the rule adds the imports for the collection types in strawman, the scala imports are still left in the source files. This basically remove all the global imports of scala's collections to avoid naming clashes. This is not perfect and needs improvement to handle imports such as:

`import collection._`
or 
```
import scala.collection
import collection.mutable
```

Reply:
after thinking a bit more about this, maybe a better approach would be to rewrite the imports to the corresponding packages in strawman. Then in `replaceSymbols` instead of having all collection types in there we would keep only the special cases (like predef, LazyList, Traversable, ...). This would likely avoid that local imports become global imports. I can dig more on this before proceeding with this PR